src/modules/dashboard/components/CreateEditDomainDialog/CreateEditDomainDialog.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { FormikProps } from 'formik';
+import * as yup from 'yup';
+
+import Dialog, { DialogProps } from '~core/Dialog';
+import { ActionForm } from '~core/Fields';
+
+import { Colony } from '~data/index';
+import { ActionTypes } from '~redux/index';
+import { WizardDialogType } from '~utils/hooks';
+
+import DialogForm from './CreateEditDomainDialogForm';
+
+export interface FormValues {
+  domainName: string;
+  domainPurpose: string;
+  annotation: string;
+}
+
+interface CustomWizardDialogProps {
+  prevStep: string;
+  colony: Colony;
+  id?: string;
+}
+
+type Props = DialogProps & WizardDialogType<object> & CustomWizardDialogProps;
+
+const displayName = 'dashboard.CreateEditDomainDialog';
```

Comment:
I would have gone with `ManageDomainsDialog`, but it's your choice

Reply:
As already mentioned on discord, `ManageDomainsDialog` already exists